### PR TITLE
Make compliant with jscs >= 1.10

### PR DESCRIPTION
**Pull request by benpatterson**
@clintonb @dsjen 

All indents. 

I'm also encountering an error in my local environment with the gulp-jscs plugin, so I'm curious what Travis has to say about that. But, this fixes the lint errors. Pls have a look?

### CODE REVIEW

File: analytics_dashboard/static/js/enrollment-demographics-age-main.js [hunk 1]
```diff
@@ -10,34 +10,34 @@ require(['vendor/domReady!', 'load/init-page'], function(doc, page) {
         function (_, DataTableView, HistogramView) {
 
         // used in the table to show ages above this are binned--displayed as "100+"
```
Comment (clintonb) [inline analytics_dashboard/static/js/enrollment-demographics-age-main.js]: Indent.

File: analytics_dashboard/static/js/enrollment-demographics-age-main.js [hunk 2]
```diff
@@ -10,34 +10,34 @@ require(['vendor/domReady!', 'load/init-page'], function(doc, page) {
...
+                modelAttribute: 'ages',
+                excludeData: ['Unknown'],
+                trends: [{
+                    title: gettext('Number of Students'),
+                    color: 'rgb(58, 162, 224)',
+                    maxNumber: maxNumber
+                }],
+                x: { key: 'age' },
+                y: { key: 'count' },
             // Translators: <%=value%> will be replaced with an age.
```
Comment (clintonb) [inline analytics_dashboard/static/js/enrollment-demographics-age-main.js]: Indent.

**Comment by benpatterson** [comment]
@clintonb @dsjen Looking better now...

**Comment by clintonb** [comment]
:+1:

**Comment by dsjen** [comment]
Thank you for tackling that!  Looks good to me.

**Pull request merged**